src/Spectre.libGenetic/MaximalFillupCondition.cpp

```diff
@@ -27,11 +28,7 @@ MaximalFillupCondition::MaximalFillupCondition(size_t minimalFillup, std::unique
 
 bool MaximalFillupCondition::checkCurrentCondition(const spectre::algorithm::genetic::Individual &individual)
 {
-    auto number = 0u;
-    for (bool value : individual.getData())
-    {
-        number += value;
-    }
+    size_t number = std::accumulate(individual.getData().begin(), individual.getData().end(), 0u);
```

Comment:
`number`? What *number*? Try to be more expressive.

Reply:
done